From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Sat, 5 Apr 2014 13:18:23 +0100
Subject: build Doxygen

This patch causes the build system to generate refman.tex.
The same effect can be had by adding the command
 $(MAKE) -C obj-* doc
to the debian/rules override_dh_auto_build target.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4c3ae5b40..e94ac596a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -418,7 +418,7 @@ if (DOXYGEN_FOUND)
   )
 
   # Generate documentation.
-  add_custom_target(doc
+  add_custom_target(doc ALL
       COMMAND "${DOXYGEN_EXECUTABLE}" "${CMAKE_BINARY_DIR}/Doxyfile"
       DEPENDS "${CMAKE_BINARY_DIR}/Doxyfile"
       WORKING_DIRECTORY "${CMAKE_BINARY_DIR}"
